.gitignore

```diff
@@ -2,3 +2,6 @@ subprojects/*
 !subprojects/*.wrap
 .pixi/
 .cache/
+
+# CMake build directory
+/build/
```

Comment:
I feel like, someone more familiar with CMake please correct me if I'm wrong, this has been the common way of ignoring build directories with CMake. CMake doesn't produce a `.gitignore` in its build directory, which I think is because of support for in-source build. I've seen people hacking to make CMake do what meson does, see [this](https://www.scivision.dev/cmake-auto-gitignore-build-dir/), but I personally think that's uglier than adding it to gitignore.

Reply:
I personally see ignoring `build/` as slightly more clutter than needed, but ultimately benign clutter. I don't feel strongly either, but I think I'd like to see PRs moving along.

Reply:
What would be an alternative to ignoring that directory? Does cmake has an option to ignore its own out of source build directory?

Reply:
Oh I didn't realize I could define user-wide ignores. Then I'm guessing the `binaryDir` setting in `CMakePresets.json` also wouldn't be necessary then?